Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/Filters/RevitViewsFilter.cs

```diff
@@ -125,4 +141,37 @@ public void SetContext(RevitContext revitContext)
     _revitContext = revitContext;
     _doc = _revitContext.UIApplication?.ActiveUIDocument.Document;
   }
+  
+  // NOTE: Element collector returns parts and source elements even when Parts Visibility is set as "Show Parts" only.
+  // Below function collects list of ids to exclude from final list.
+  private HashSet<ElementId> GetSourceElementIdsToExclude(IEnumerable<Element> elements)
+  {
+    var elementsToExclude = new HashSet<ElementId>();
+
+    foreach (var element in elements)
```

Comment:
I was initially weary about additional `foreach` loop. Tested LINQ expressions and other native Revit API calls, but this method remained most performant! Let's keep as it, nice!